taskerpluginlibrary/src/main/java/com/joaomgcd/taskerpluginlibrary/runner/TaskerPluginRunner.kt

```diff
@@ -47,6 +47,11 @@ abstract class TaskerPluginRunner<TInput : Any, TOutput : Any> {
         TaskerPluginRunner.startForegroundIfNeeded(this, notificationProperties)
     }
 
+    @TargetApi(Build.VERSION_CODES.O)
+    fun startForegroundIfNeeded(intentServiceParallel: IntentServiceParallel) {
```

Comment:
I did not use an extension function here as it would clash with `IntentServiceParallel.startForegroundIfNeeded`